src/components/state-history-chart-timeline.html

```diff
@@ -41,39 +41,22 @@
   },
 
   drawChart: function () {
-    var root = Polymer.dom(this);
     var stateHistory = this.data;
-    var chart;
     var dataTable;
     var startTime;
     var endTime;
-    var numTimelines;
-    var format;
-    var daysDelta;
 
     if (!this.isAttached) {
       return;
     }
 
-    while (root.node.lastChild) {
-      root.node.removeChild(root.node.lastChild);
-    }
 
-    if (!stateHistory || stateHistory.length === 0) {
+    if (!this.isAttached) {
       return;
     }
 
-    chart = new window.google.visualization.Timeline(this);
-    dataTable = new window.google.visualization.DataTable();
-
-    dataTable.addColumn({ type: 'string', id: 'Entity' });
-    dataTable.addColumn({ type: 'string', id: 'State' });
-    dataTable.addColumn({ type: 'date', id: 'Start' });
-    dataTable.addColumn({ type: 'date', id: 'End' });
-
-    function addRow(entityDisplay, stateStr, start, end) {
-      var stateDisplay = stateStr.replace(/_/g, ' ');
-      dataTable.addRow([entityDisplay, stateDisplay, start, end]);
+    if (!stateHistory || stateHistory.length === 0) {
```

Comment:
Can merge this if with the one above.

Reply:
Actually, this check is now wrong because you removed the code that made the div all empty. Now if the new data is empty, it will not remove the old one and thus show stale data